client/src/components/layout/TripShow.js

```diff
@@ -0,0 +1,36 @@
+import React, { useState, useEffect } from "react"
+
+const TripShow = (props) => {
+  const [trip, setTrip] = useState({})
+  const id = props.match.params.id
```

Comment:
We can add a line between 4&5, if there were multiple pieces of state we would want to group them together for example